packages/math-input/src/components/keypad/keypad.stories.tsx

```diff
@@ -1,8 +1,9 @@
 import {action} from "@storybook/addon-actions";
 import {INITIAL_VIEWPORTS} from "@storybook/addon-viewport";
+import {ComponentStory} from "@storybook/react";
 import * as React from "react";
 
-import Keypad from "./index";
+import Keypad, {Props as KeypadProps} from "./index";
```

Comment:
I don't think it's enforced in the linter yet, but I think we're still trying to use `import SomeComponent` and `import type SomeType`

Reply:
In TypeScript land, we [don't need to use](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html) the `type` keyword on imports or exports.  We can just `import {SomeType} ...`  

Also, it's a bit of a "mouthful" but we no longer need to be importing Props types, but rather we can derive the props types from the component itself using:

```js
JSX.LibraryManagedAttributes<
    typeof MyComponent,
    React.ComponentProps<typeof MyComponent>
>
```

https://khanacademy.atlassian.net/wiki/spaces/ENG/pages/2201682693/TypeScript+for+Flow+Developers#React-Types

Reply:
Oh because that is so much easier to understand 😕 ?

Reply:
Oh @jeremywiebe , I think you meant `React.ComponentProps<typeof MyComponent>` which is much nicer

Reply:
Well okay, so we use this big, impossible-to-read-and-harder-to-remember syntax clearly meant for internal use every time we need access to prop types? You got it.